src/main/java/com/gtnewhorizons/angelica/proxy/ClientProxy.java

```diff
@@ -76,9 +81,23 @@ public void init(FMLInitializationEvent event) {
             FMLCommonHandler.instance().bus().register(Iris.INSTANCE);
             MinecraftForge.EVENT_BUS.register(Iris.INSTANCE);
         }
+
+
+        FMLCommonHandler.instance().bus().register(this);
+        MinecraftForge.EVENT_BUS.register(this);
+
+        glsmKeyBinding  = new KeyBinding("Print GLSM Debug", Keyboard.KEY_GRAVE, "Angelica Keybinds");
+        ClientRegistry.registerKeyBinding(glsmKeyBinding);
     }
 
-    @Override
+    @SubscribeEvent
+    public void onKeypress(InputEvent.KeyInputEvent event) {
```

Comment:
I'm not sure if this event is fired in GUIs. If it isn't then we may want to use `Keyboard` directly instead, so it can be used to debug issues in them.